lib/contentful/migration.ts

```diff
@@ -3,14 +3,28 @@ import { runMigration } from "contentful-migration"
 import { info, success } from "../logger"
 import { MigrationOptions, PendingMigration } from "../types"
 
+// eslint-disable-next-line @typescript-eslint/member-delimiter-style
```

Comment:
Can we avoid sprinkling this all over the code?
If you ask me, disable comments should be accompanied by an explanation
If it has too many false positives, it should be disabled entirely.

Reply:
That is actually not needed, I forgot to remove it. 

I just removed the comment.

Reply:
There is a second one. Could we do this in general?

Reply:
Thanks 🙏